tests/zkevm/test_worst_compute.py

```diff
@@ -757,14 +757,14 @@ def test_worst_jumps(
     """Test running a JUMP-intensive contract."""
     env = Environment()
 
-    def jump_seq():
-        return Op.JUMP(Op.ADD(Op.PC, 1)) + Op.JUMPDEST
+    def jump_seq(dest: int):
+        return Op.JUMP(Op.PUSH2(dest)) + Op.JUMPDEST
```

Comment:
I think this test should be written as `Op.JUMPDEST + Op.JUMP(0)` (no need for a loop) :smile: :+1: